content/22.vaccines.md

```diff
@@ -669,9 +678,14 @@ Additionally, Germany has stated that if Sputnik V were approved by EMA, it woul
 Russia is developing other vaccine candidates and has approved a third vaccine, CoviVac, which is an inactivated vaccine produced by the Chumakov Centre in Moscow, despite the fact the clinical trials have yet to begin [@url:https://www.reuters.com/article/us-health-coronavirus-russia-vaccine-idUSKBN2AK07H].
 
 In Asia, China and India are the main COVID-19 vaccination developers and providers.
+Thus far, over 63 million people have been vaccinated against COVID-19 vaccine in India [@url:https://www.mygov.in/covid-19/].
+A broad range of COVID-19 vaccine candidates are under investigation in order to respond to the COVID-19 pandemic.
 In India, the Covaxin vaccine produced by Bharat Biotech received emergency authorization on January 3rd, 2021, despite the lack of phase III data until March 3rd [@url:india-covid-19-vaccine.html; @url:https://www.bharatbiotech.com/images/press/covaxin-phase3-efficacy-results.pdf].
 Following the release of the phase III data indicating 81% efficacy, Zimbabwe authorized the use of Covaxin [@url:http://www.xinhuanet.com/english/2021-03/04/c_139783893.htm].
-In February, 2021, Bharat Biotech received approval from Indian officials to commence a phase I study of an intranasal chimpanzee-adenovirus (ChAd) vectored SARS-CoV-2-S vaccine called BBV154 [@url:https://www.bharatbiotech.com/intranasal-vaccine.html].
+In addition to COVAXIN®, Bharat Biotech has also developed an adenovirus vector-based intranasal BB154 vaccine candidate that induces neutralizing IgG, mucosal IgA, and T cell responses [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
```

Comment:
Is BB154 missing a V or is this a different vaccine than the line below but with a very close name?

Reply:
Sorry, missed the V in BBV154

Reply:
```suggestion
In addition to COVAXIN®, Bharat Biotech has also developed an adenovirus vector-based intranasal BBV154 vaccine candidate that induces neutralizing IgG, mucosal IgA, and T cell responses [@url:https://www.bharatbiotech.com/intranasal-vaccine.html]. 
```

Reply:
Added!

Reply:
Yay! I was hoping that was the case because my eyes aren't shart enough to handle two names that close 😆